wildfly-swarm/redhat/cache/booster.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+  runsOn: 
+    - '!starter'
+    - '!osio'
+name: WildFly Swarm - Cache
+description: Demonstrate how to use a cache server in WildFly Swarm
```

Comment:
That is correct, they are no longer used in the new UI

Reply:
What about the old UI? We didn't switch to the new one everywhere yet, did we?

Reply:
The only place where this information is read is in the OSIO legacy UI, and this booster won't be listed there, so it's safe to remove

Reply:
Although it would be interesting to keep it for the logs: https://ci.centos.org/job/devtools-launcher-booster-catalog/45/console displays what the booster is about